pkg/fqdn/matchpattern/matchpattern.go

```diff
@@ -88,6 +93,12 @@ func ToAnchoredRegexp(pattern string) string {
 func ToUnAnchoredRegexp(pattern string) string {
```

Comment:
function doc comment should needs to learn about `"**"` too

Reply:
Added